logging/tests/unit/handlers/transports/test_background_thread.py

```diff
@@ -166,15 +166,17 @@ def test_constructor(self):
         grace_period = 50
         max_batch_size = 50
         max_latency = 0.1
+        include_logger_name = True
 
         worker = self._make_one(
             logger, grace_period=grace_period, max_batch_size=max_batch_size,
-            max_latency=max_latency)
+            max_latency=max_latency, include_logger_name=include_logger_name)
 
         self.assertEqual(worker._cloud_logger, logger)
         self.assertEqual(worker._grace_period, grace_period)
         self.assertEqual(worker._max_batch_size, max_batch_size)
         self.assertEqual(worker._max_latency, max_latency)
+        self.assertEqual(worker._include_logger_name, include_logger_name)
```

Comment:
Just assert that it is true, e.g.:

```suggestion
        self.assertTrue(worker._include_logger_name)
```

Reply:
Done!